README.md

```diff
@@ -86,32 +86,10 @@ class App extends Component {
 
 ## Development Setup
 
-You can use `npm link` to link your local copy of the `react-plotly.js-editor` to your test repo.
-
-With `npm link` you can get some errors related to
-[multiple copies of react being loaded](https://github.com/facebookincubator/create-react-app/issues/1107).
-To get around this, you can create an
-[alias](https://github.com/facebookincubator/create-react-app/issues/393) that
-points your project to the copy of react that it should be using or you can
-simply remove `react` and `react-dom` from `react-plotly.js-editor`.
-
-### Using the `simple` example for development
-
-In one terminal:
-
-```
-npm link       # registers react-plotly.js-editor for subsequent npm link calls
-npm run watch  # keep this running
 ```
-
-In another terminal:
-
-```
-cd examples/simple # or any other application directory
 npm install
-rm -rf node_modules/react node_modules/react-dom node_modules/react-plotly.js-edit
-npm link react-plotly.js-edit
-npm start # keep this running
+npm start
+# start hacking
 ```
```

Comment:
haha, well, yeah, a lot better
all the better that we do this before Jack releases all the marketing on this.

That way, if we're so lucky that others can't wait to contribute, well, there's an easier way to set all this up! Thanks!

Reply:
Would you use this for development going forward or is it still missing something that would keep you using the simple example with npm-link?

Reply:
yes, let's use this going forward! thanks!